docs/qcoverage/index.md

```diff
@@ -44,68 +44,91 @@ there is always one thread in addition to the number specified with
+--type              Req, the type of coverage to perform. Possible Values: [sequence,
+                      physical]. 
+--validation        Opt, how strict to be when reading a SAM or BAM. Possible values:
+                      [STRICT, LENIENT, SILENT].
+--version           Show version number.
 ~~~~
 
-For physical read depth:
+## Output
 
-~~~~{.text}
-java -jar qcoverage.jar -t phys \
-    --bam test1.bam --gff3 GRCh37.gff3 -o report.vcf --log logfile
-~~~~
+### standard mode output
 
+The default output is tab delimited text format, with or without option "--output-format TXT". Here file extension ".txt" is automatically append to putput file. 
 
+~~~~
+]$java -jar qcoverage.java --type physical --output /path/report \
+--input-gff3 /path/GRCh37_ICGC_standard_v2.gff3 --input-bam /path/input.bam --log /path/output.log 
 
-## Options
+]$less /path/report.txt
 
-~~~~{.text}
---help, -h      Shows this help message.               
---version, -v   Print version.                    
---bam           Input BAM file.
---bai           Index file for --bam.
---gff3          The GFF3 file defining the features.   
---log           File where log output will be directed (must have write permissions).       
---loglevel      Logging level [INFO, DEBUG], Def=INFO. 
--n              Opt, Number of worker threads (yields n+1 total threads), Def=1.
---output, -o    Req, Output file.
---per-feature   Opt, Perform per-feature coverage.
---query, -q     Opt, The query string for selecting reads for coverage.
---type, -t      The type of coverage to perform [seq, sequence, phys, physical].
---xml           Opt, Output report in XML format.
---vcf           Opt, Output report in VCF format. Needs the per-feature flag to also be set.
+#coveragetype   featuretype     numberofbases   coverage
+physical        chrom   2518007876      0x
+physical        chrom   385426364       1x
+...
+physical        chrom   2       1539x
+physical        chrom   6       1540x
+~~~~
+
+You can also specify the output to be XML format. Here file extension ".xml" is automatically append to putput file. 
+~~~~
+]$java -jar qcoverage.java --type physical \
+--output-format XML --output /path/report \
+--input-gff3 /path/GRCh37_ICGC_standard_v2.gff3 --input-bam /path/input.bam --log /path/report.log 
+
+]$less /path/report.xml
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<QCoverageStats>
+    <coverageReport feature="chrom" type="PHYSICAL">
+        <coverage bases="2518007876" at="0"/>
+        <coverage bases="385426364" at="1"/>
+...
 ~~~~
 
-## VCF output
 
-It is possible to have output from the --per-feature mode in either XML or
-VCF format. The XML format is very extensive with an XML table for each
+### per-feature mode output
+It is possible to have output from the --per-feature mode in either TXT, XML or
+VCF format. The TXT and XML format are very extensive with an XML table for each
```

Comment:
Does the TXT format have an XML table for each feature?

Reply:
The Txt file  looks like: 
`#GL000232.1     GRCh37_ICGC_standard_v2.properties      chrom   1       40652   .       .       .       FASTA_byte_offset=3145366215
physical        22608   0x
`
the XML looks like:
` <coverageReport feature="GL000232.1 GRCh37_ICGC_standard_v2.properties      chrom   1       40652   .       .       .       FASTA_byte_offset=3145366215" type="PHYSICAL">
        <coverage bases="22608" at="0"/>
   `
both same data but different formate

Reply:
OK, the sentence implies that there is an XML table for both the TXT and XML formats. Could this be re-worded?

Reply:
fixed